### DISCUSSION TITLE
Expected behaviour for action Github.Open.In.Browser?

### DISCUSSION THREAD

**Discussion by DamonBakerANZ** [JetBrains/ideavim#887] [General]
Hi, I was going to file a bug report but I thought it would be better to first open a discussion in case I'm misunderstanding the correct behaviour.

In normal mode, if I run the action "Open on Github" the behaviour is as you would expect, a browser window is opened which directs to that file. Visual mode is similar, but with the range of line numbers also highlighted in the URL.

![CleanShot 2024-05-17 at 15 12 53@2x](https://github.com/JetBrains/ideavim/assets/130536422/95c02212-963d-4cd1-8d3e-f34c4ae37d03)

![CleanShot 2024-05-17 at 15 01 49@2x](https://github.com/JetBrains/ideavim/assets/130536422/c5e92905-533e-4529-bb92-824ddcad6e18)

However, if I execute the same command through a mapping like `map <Leader>go <Action>(Github.Open.In.Browser)` or running the command directly with `:action Github.Open.In.Browser`, all that happens is this dialog appears. This happens regardless of whether I am in normal mode or visual.

![CleanShot 2024-05-17 at 15 09 21@2x](https://github.com/JetBrains/ideavim/assets/130536422/4470c0b4-075e-4744-917b-bfec96771119)

Does this look like it might be a bug? Or do I have my mapping set incorrectly?

**Comment by citizenmatt** [comment]
This is a known issue, unfortunately: [VIM-3422](https://youtrack.jetbrains.com/issue/VIM-3422/Ideavim-actions-calling-Github-and-GitLab-plugins-return-Nothing-here)

**Comment by Kuresov** [comment]
Issue looks to be tracked here now: [VIM-3447](https://youtrack.jetbrains.com/issue/VIM-3447/Open-on-github-doesnt-work-with-ideavim-action-bindings)

**Comment by helmerdx** [comment]
Some news about this. I want to use this often

**Comment by piacsek** [comment]
It's working again to me after I updated to `2.20.3` 🎉 😻

**Comment by helmerdx** [comment]
Yes, working again in `2.21` and even recognizes highlighting of lines
